dbt/include/greenplum/dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 config-version: 2
 name: dbt_greenplum
-version: 1.0
+version: 1.0 #im usure if i need to change this? 
```

Comment:
No you don't need to change it. It is about version of certain project
https://docs.getdbt.com/docs/build/projects

Reply:
okay thanks ill remove the comment